docs/competition/livestream/README.md

```diff
@@ -0,0 +1,29 @@
+# Livestream
+
+Some events, such as the [competition](../README.md) online-only events are livestreamed.
+
+## Platforms
+
+Events are streamed to the Student Robotics [YouTube](https://www.youtube.com/user/studentrobotics) account.
+
+## Software
+
+We use [OBS Studio](https://obsproject.com/) as the livestreaming software used to transmit the video feeds to YouTube. OBS deals with mixing, scene transmission and screen capture.
+
+OBS configuration is split into 2 fundamental objects: Scenes and Sources. A scene is the thing shown on screen, which is made up of multiple sources.
+
+## Hardware
+
+The hardware requirements for hosting a livestream are not large, however a stable internet connection is a must. A dedicated GPU is useful to ensure the system isn't taxed too heavily.
+
+## Music
+
+As background music during livesteams, we'll often play quiet music.
+
+For finals events, more dramatic music is played vs other sessions, to emphasize the importance. For finals events, we have heavily used [Monstercat](https://www.monstercat.com/) in the past, and for other events [incompetech](https://incompetech.com/), as these sources can be used without purchasing a licence, provided the required attribution is given.
```

Comment:
Might be worth mentioning that you have to request a free licence for Monstercat but incompetech is free to use anytime

Reply:
This isn't actually true. I only contacted Monstercat as their terms were a bit unclear. Incompetech is royalty free, Monstercat just take a cut of your advertising money, which given we don't run ads and our audience is so low, is 0.

Reply:
Ah ok that makes sense!